changelog/@unreleased/pr-684.v2.yml

```diff
@@ -0,0 +1,16 @@
+type: break
+break:
+  description: "## Before this PR\n`gitVersion()` or `versionDetails()` does not have
```

Comment:
I would also leave out the "Before this PR" bits and just have a couple of bullet points, one for the change and one describing the break.